doc/2/plugins/plugin-context/accessors/sdk/index.md

```diff
@@ -69,13 +69,70 @@ async myAwesomePipe (request) {
 
 - The created document will have the `author` metadata property set to `null`.
 
+## Realtime notifications
+
+<SinceBadge version="change-me" />
+
+The [realtime](/sdk/js/7/controllers/realtime) is entirely available with the [subscribe](/sdk/js/7/controllers/realtime/subscribe) and [unsubscribe](/sdk/js/7/controllers/realtime/unsubscribe) methods.  
+
+Realtime subscription should be made in the plugin [init](core/2/plugins/guides/manual-setup/init-function) method or in a hook on the [kuzzle:start:before](/core/2/plugins/guides/events/kuzzle-start) event.
+
+::: warning
+You should avoid making subscription at runtime because that can lead to unwanted behavior since the subscriptions will not be replicated on every cluster node.
+:::
+
+The `propagate` option defines if, for that subscription, notifications should be propagated to (and processed by) all cluster nodes, or if only the node having received the triggering event should handle it.
+
+By default When you receive a notification, only one node will execute the associated callback.  
+
+#### propagate: false (default)
+
+With `propagate: false`, the callback function will be executed only on the node receiving a request who triggers a notification. (only one execution) 
+
+::: info 
+This behavior is suitable for most usage like sending emails, write in the database, call an external API, etc.
+:::
+
+#### propagate: true
+
+With `propagate: true`, the callback function will be executed one each node of the cluster. (n executions)
```

Comment:
```suggestion
With `propagate: true`, notifications are propagated to all nodes of a cluster, executing all callback functions.
```

Reply:
That's not true, the notifications are always propagated but they will not trigger notifications but since it's an internal behavior I'm OK with this sentence if it's more easy to understand for the users